__tests__/app.test.js

```diff
@@ -0,0 +1,30 @@
+const request = require("supertest");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const db = require("../db/connection");
+const { topicData, userData, articleData, commentData } = require("../db/data/test-data/index");
+
+beforeEach(() => seed({ topicData, userData, articleData, commentData }));
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("GET /api/topics", () => {
+  it("status 200, should return an array of topic objects, each of which should have the following properties:slug ,description", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        const { topics } = body;
+        topics.forEach((topic) => {
+          expect(topic).toHaveProperty("slug", expect.any(String));
+          expect(topic).toHaveProperty("description", expect.any(String));
+        });
+      });
+  });
+
+  test("404, wrong url address", () => {
```

Comment:
Good test but it needs to be in a separate describe block because it is a general test - not specific to this endpoint 😄